From cc2717f8783dc02960af0bfee8a89768863bb3be Mon Sep 17 00:00:00 2001
From: Rico Tzschichholz <ricotz@ubuntu.com>
Date: Fri, 11 Jan 2019 23:41:54 +0100
Subject: [PATCH 10/19] g-i: Annotate "callback" type of
 clutter_binding_pool_override_action()

(cherry picked from commit bb3f2de58b20eb25c83990853ee2f6e7a1acf5d2)
---
 clutter/clutter-binding-pool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clutter/clutter-binding-pool.c b/clutter/clutter-binding-pool.c
index 66959c784..a76644826 100644
--- a/clutter/clutter-binding-pool.c
+++ b/clutter/clutter-binding-pool.c
@@ -588,7 +588,7 @@ clutter_binding_pool_install_closure (ClutterBindingPool  *pool,
  * @pool: a #ClutterBindingPool
  * @key_val: key symbol
  * @modifiers: bitmask of modifiers
- * @callback: function to be called when the action is activated
+ * @callback: (type Clutter.BindingActionFunc): function to be called when the action is activated
  * @data: data to be passed to @callback
  * @notify: function to be called when the action is removed
  *   from the pool
-- 
2.20.1

